peer/store-utils.go

```diff
@@ -11,12 +11,21 @@ import (
 	"github.com/gluster/glusterd2/store"
 
 	log "github.com/Sirupsen/logrus"
+	"github.com/pborman/uuid"
 )
 
 const (
 	peerPrefix string = store.GlusterPrefix + "peers/"
 )
 
+var (
```

Comment:
Can't we name them like 'MockFunctionName' with naming convention, at least I used it in my earlier commits.